src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -644,12 +654,19 @@ class AddItemsPage extends Component {
       buttons: [
         {
           label: this.props.translate('react.default.yes.label', 'Yes'),
-          onClick: onConfirm,
+          onClick: () => {
+            this.didUserConfirmAlert = true;
+          },
         },
         {
           label: this.props.translate('react.default.no.label', 'No'),
         },
       ],
+      afterClose: () => {
+        if (this.didUserConfirmAlert) {
+          onConfirm();
+        }
+      },
```

Comment:
nice workaround. Why was bumping up needed? Is it because `afterClose` is not available in lower versions?